warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -46,7 +32,7 @@ models:
             - severity
 
   - name: int_gtfs_quality__no_schedule_validation_errors
-    tests: &stg_gtfs_guideline_tests
+    tests: &stg_gtfs_guideline_tests_schedule
```

Comment:
Just confirming that these are the same changes from #2018

Reply:
yep, same changes. I keep doing this with PRs, where I should probably just make it one PR so I'm not doing the same changes twice and needing to merge one and then merge into the other.